bundle/run/job.go

```diff
@@ -317,6 +317,25 @@ func (r *jobRunner) Cancel(ctx context.Context) error {
 	return errGroup.Wait()
 }
 
+func (r *jobRunner) Restart(ctx context.Context, opts *Options) (output.RunOutput, error) {
```

Comment:
The problem then is we don't have the `runId` for the run triggered by the continuous job configuration and cannot provide live updates about the job status on the command line.

I consulted the jobs team on this, and the `/run-now` API is explicitly designed to cancel existing runs for continuous jobs to avoid this race condition.